planning/planning_scene_monitor/include/moveit/planning_scene_monitor/planning_scene_monitor.h

```diff
@@ -411,14 +411,9 @@ class PlanningSceneMonitor : private boost::noncopyable
   planning_scene::PlanningSceneConstPtr scene_const_;
   planning_scene::PlanningScenePtr      parent_scene_; /// if diffs are monitored, this is the pointer to the parent scene
   boost::shared_mutex                   scene_update_mutex_; /// mutex for stored scene
-  ros::Time                             last_update_time_; /// Last time the scene was updated
-  ros::Time                             last_robot_motion_time_; /// Last time the robot has moved
-  bool                                  enforce_next_state_update_;
 
```

Comment:
On Thu, Jul 21, 2016 at 04:13:47PM -0700, Robert Haschke wrote:
> I thought that adding new members in the end is possible, isn't it?
> This individual commit moves all new members to the end.

No it is not.
If you append new members at the end this keeps the current class layout valid,
so thus code that uses only the already existing variables of given class instances
does not break.

But still, old code that allocates instances of the class does not allocate space
for the new members. As we do not provide a factory for the PSM, we don't encapsulate
allocation in the library. So if you access the new members in the library when the
PSM was created by user-code, this *might* segfault.

The other issue with extending classes is layout-change in the arguments of function calls
or other classes. So if anyone passes a PSM as a function parameter (that is not the last one)
or uses a PSM without a pointer as a member of a class, this will break.

> Which member is renamed? Did I missed something?

You renamed `last_state_update` to `wall_last_state_update`


v4hn